spec/controllers_spec/posts_controller_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+require 'factory_bot_rails'
+
+RSpec.describe 'Posts', type: :request do
+  before(:each) do
+    # Clean up the test database before each test
```

Comment:
- [ ] Kindly consider removing all the comments that are not needed in production from your code base, these comments are useful in the development process but they should be removed when you push your code to GitHub